ankihub/main/note_type_management.py

```diff
@@ -33,8 +34,41 @@ def add_note_type(ah_did: uuid.UUID, note_type: NotetypeDict) -> NotetypeDict:
     return new_note_type
 
 
-def update_note_type_fields(note_type: NotetypeDict, fields: List[str]) -> None:
-    print("update_note_type_fields", note_type["name"], fields)
+def add_note_type_fields(
+    ah_did: uuid.UUID, note_type: NotetypeDict, new_field_names: List[str]
+) -> NotetypeDict:
+    client = AddonAnkiHubClient()
+
+    db_note_type = ankihub_db.note_type_dict(ah_did, note_type["id"])
```

Comment:
It is pythonic to use `.get()` when accessing a variable on a dict and `dict[key]` when attributing a value. `dict[key]` may raise `KeyError`

```suggestion
    db_note_type = ankihub_db.note_type_dict(ah_did, note_type.get("id"))
```

Reply:
`id` is guaranteed to exist here. It's also probably better to fail with `KeyError` rather than potentially mask a bug if `ankihub_db.note_type_dict()` is OK with `note_type_id=None`.